account-kit/smart-contracts/src/ma-v2/account/account.ts

```diff
@@ -120,8 +129,36 @@ export async function createSMAV2Account(
     ),
   });
 
+  // TODO: add deferred action flag
+  const getAccountNonce = async (nonceKey?: bigint): Promise<bigint> => {
+    // uint32 entityId + (bytes1 options) makes a uint40
+    const nonceSuffix: bigint =
+      entityId * 256n + (isGlobalValidation ? 1n : 0n);
+
+    if (nonceKey) {
+      // if nonceKey is less than nonceSuffix it cannot contain it
+      if (nonceKey < nonceSuffix || nonceKey % 2n ** 40n != nonceSuffix) {
```

Comment:
⚠️ **[eslint]** <[eqeqeq](https://eslint.org/docs/latest/rules/eqeqeq)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected '!==' and instead saw '!='.
<!-- __reviewdog__:ChA3MDRkYzMyNTI0ZDgwZWM2EgZyZGpzb24= -->